webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/DataGridContextMenu/DataGridContextMenuCellEditingService.ts

```diff
@@ -140,13 +159,7 @@ export class DataGridContextMenuCellEditingService {
           ACTION_DATA_GRID_EDITING_REVERT_SELECTED_ROW,
         ].includes(action);
       },
-      getActionInfo(context, action) {
-        if (action === ACTION_EDIT) {
-          return { ...action.info, label: 'data_grid_table_editing_open_inline_editor', icon: 'edit' };
-        }
-
-        return action.info;
-      },
+      getActionInfo: this.getActionInfo.bind(this),
```

Comment:
```suggestion
      getActionInfo: (context, action) => {
        if (action === ACTION_EDIT) {
          return { ...action.info, label: 'data_grid_table_editing_open_inline_editor', icon: 'edit' };
        }

        return action.info;
      },
```
just use arrow function